moto/ec2/models.py

```diff
@@ -299,6 +299,7 @@ def __init__(self, ec2_backend, image_id, user_data, security_groups, **kwargs):
         self.subnet_id = kwargs.get("subnet_id")
         self.key_name = kwargs.get("key_name")
         self.source_dest_check = "true"
+        self.launch_time = datetime.now().isoformat()
```

Comment:
I don't think it matters much in this case, only that the time is in the correct format.